vault/Gottesman-Knill theorem.md

```diff
@@ -0,0 +1,8 @@
+#simulation #stabilizer
+
+Any quantum computation composed only by Clifford gates can be simulated efficiently by a classical computer following the stabilizer formalism. The key is to update Pauli operators according to the rules imposed by Clifford gates, from the [[Clifford group]]. This works only if the initial state is stabilized by a collection of Pauli strings. 
```

Comment:
stabilized by a 'collection of Pauli strings' -> 'maximal commutative subgroup of the Pauli group'